alma/tests/Unit/Helper/CarrierHelperTest.php

```diff
@@ -24,11 +24,72 @@
 
 namespace Alma\PrestaShop\Tests\Unit\Helper;
 
+use Alma\PrestaShop\Builders\CarrierHelperBuilder;
+use Alma\PrestaShop\Helpers\CarrierHelper;
+use Alma\PrestaShop\Model\CarrierData;
 use PHPUnit\Framework\TestCase;
 
 class CarrierHelperTest extends TestCase
 {
     public function testGetParentCarrierNameById()
```

Comment:
I think we can use dataProvider to make a better refacto of this test

Reply:
I think there is too many different Mock to do that, the tests are very different